### DISCUSSION TITLE
Can't start rtl server

### DISCUSSION THREAD

**Discussion by onkleklaus** [Ride-The-Lightning/RTL#1389] [Troubleshooting Questions] [answered]
Hi, following the raspibolt guide I've installed rtl but attempting to run it this error appears:

```
ERROR: Config => Something went wrong while configuring the node server:
SyntaxError: Bad escaped character in JSON at position 27.

file:///home/rtl/RTL/backend/utils/config.js:369
                throw new Error(err);
                      ^

Error: SyntaxError: Bad escaped character in JSON at position 27
    at ConfigService.setServerConfiguration (file:///home/rtl/RTL/backend/utils/config.js:369:23)
    at new ConfigService (file:///home/rtl/RTL/backend/utils/config.js:372:14)
    at file:///home/rtl/RTL/backend/utils/config.js:375:23
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:123:5)

Node.js v20.13.1
```
My RTL-Config.json:

```
{
  "multiPass": "mypassword",
  "port": "3000",
  "defaultNodeIndex": 1,
  "dbDirectoryPath": "",
  "SSO": {
    "rtlSSO": 0,
    "rtlCookiePath": "",
    "logoutRedirectLink": ""
  },
  "nodes": [
    {
      "index": 1,
      "lnNode": "Node 1",
      "lnImplementation": "LND",
      "Authentication": {
        "macaroonPath": "/home/rtl",
        "configPath": "/data/lnd/lnd.conf",
        "swapMacaroonPath": "",
        "boltzMacaroonPath": ""
      },
      "Settings": {
        "userPersona": "MERCHANT",
        "themeMode": "DAY",
        "themeColor": "PURPLE",
        "channelBackupPath": "",
        "logLevel": "ERROR",
        "lnServerUrl": "https://127.0.0.1:8080",
        "swapServerUrl": "https://127.0.0.1:8081",
        "boltzServerUrl": "https://127.0.0.1:9003",
        "fiatConversion": false,
        "unannouncedChannels": false
      }
    }
  ]
}


```
Any help will be appreciated

**Comment by ShahanaFarooqui** [comment, marked as answer]
According to the error message, your RTL-Config.json contains certain escaped characters and removing that incorrectly escaped character or a stray backslash from the position should fix it.

But your above config is already valid and it worked for me perfectly. Are you sure this is the correct location/config file?

**Reply by onkleklaus** [reply]
Thanks for the reply.
Yes, this is my config located at
` /home/rtl/RTL/RTL-Config.json`
At the end of the installation npm has shown some vulnerabilities (1 low and 4 moderate iirc). May be those are the culprit?

**Comment by ShahanaFarooqui** [comment]
No, npm vulnerabilities have nothing to do with this error. It is definitely related to your config’s json. Can you validate your json and check if there is some special character. Also try it after removing blank macaroon and channel backup paths though it is working fine for me with blank paths also.

**Reply by onkleklaus** [reply]
Removing blank macaroon and channel backup paths doesn't fix the issue.
Attempting to validate the json got this error:

```
Invalid JSON!

Error: Parse error on line 18:
...nd/lnd.conf",      },      "Settings":
----------------------^
Expecting 'STRING', got '}'
```

edit:
Even removing the trailing comma in this line

`"configPath": "/data/lnd/lnd.conf",`

RTL server doesn't start

**Reply by ShahanaFarooqui** [reply]
Did removing the trailing comma successfully validate the json? Till the json is invalid, RTL server will keep throwing errors.

**Reply by onkleklaus** [reply]
Thanks again for your help.
I know that it seems a bulls..t but got the fix simply copy/pasting into a full screen terminal window instead of a partial screen one.
BTW the trailing comma in the configPath line is there and the server starts fine now.
Sorry for your time lost
